docs/index.md

```diff
@@ -1,23 +1,49 @@
+The proportion of the population that is vaccinated increases over the time frame, which you can see as the bubbles move from left to right. The quantity of deaths increases as well month-by-month, which makes sense: more deaths accumulate each month while the denominator (the 2019 Census population) stays the same. We also know that the Delta variant hit during this time period, which was much more contagious than previous strains: we should expect deaths to increase.  
+  
+However, consider the trajectories of the large bubbles (representing highly populous counties) vs. their smaller counterparts. The largest bubbles progress steadily into the 70-80% vaccinated range by the end of the timeframe, and their rate of deaths generally tops out at 100 deaths/100K people. Many smaller bubbles struggle to reach the 50% vaccinated mark, and several of their death rates soar into the 300-400 deaths/100K range. Comparing these two groups suggest that vaccination is highly effective--otherwise the more highly-populated areas would likely have similar (or even worse) death rates, given the physical proximity of the population in these urban centers.   
+  
+Regional differences prove to be fertile ground for continued research. Many of the small blue dots, representing counties in the South, don't reach the 50% vaccinated threshold. However, the large blue bubbles (representing urban areas in the South) follow a more direct left-to-right trajectory--and tellingly, keep their death rates low.  
+  
+These findings suggest that behavioral differences in rural vs. urban communities (mask wearing, social distancing, etc.) or differences in political ideology may affect vaccine adoption--but they offer little doubt that the COVID vaccines are effective at minimizing the death toll of the virus. 
+
+## Vaccine Effectiveness - County Comparison
+
+Another way to explore vaccine effectiveness is by comparing specific counties. Similar to the Complete U.S. visualization, the analysis below is comparing the number of deaths and vaccination percentages over the same time period. However, the visualization shows the difference between Kent County, RI and Tuscaloosa County, AL.
+
+<img src="https://github.com/ds5010/vaccines/blob/main/img/comparison.png" width="500" height="300"> |
+
+
+## Methodology -  County Comparison
+
+For this analysis, the team chose counties with similar population demographics but highly different rates of vaccination to see the impact (if any) on death rates due to COVID-19. The same JHU and CDC data sources were used.
+
+## Vaccine Effectiveness - County Comparison
+
```

Comment:
I just added an example_comparison.png file in my latest pull request - if we merge that first you could add that here.

Reply:
Yes that is a good idea, added!